### PR TITLE
add support for additional log_formats and access_logs

### PR DESCRIPTION
**Pull request by tod-uma**
I needed to be able to add additional log formats and additional access logs. This PR contains the changes I made.

### CODE REVIEW

File: templates/nginx.conf.j2 [hunk 1]
```diff
@@ -31,7 +31,14 @@ http {
 
     log_format  main  {{ nginx_log_format|indent(23) }};
 
+    {% for format_name, format in nginx_log_formats.iteritems() -%}
```
Comment (shakalandy) [inline templates/nginx.conf.j2:34]: I guess this should be a "nginx_log_formats.items", or?

**Review by shakalandy — COMMENTED** [review]
Nice one, was thinking about the same - we need json logs additional to the normal logs.
Thanks!

File: templates/nginx.conf.j2 (hunk 1 above)
Comment (tod-uma) [reply templates/nginx.conf.j2:34]: huh, it worked on my system. Looks like it's a python 2 vs python 3 issue. I'll rework it.

**Comment by shakalandy** [comment]
This one would also be pretty nice @geerlingguy and pretty straight forward.
Anything we could do to help, here? thanks!

**Pull request closed without merging**